docs/gettingstarted/index.rst

```diff
@@ -84,6 +84,14 @@ Getting started with Flocker
          .. task:: test_homebrew flocker-|latest-installable|
             :prompt: you@laptop:~$
 
+         Ubuntu 15.04
```

Comment:
Adding this makes the tabs run over onto two lines here, but this is changing in #1506 anyway.